blocks/api/raw-handling/test/integration/index.js

```diff
@@ -18,6 +18,8 @@ const types = [
 	'google-docs',
 	'ms-word',
 	'ms-word-online',
+	'one-image',
```

Comment:
Is there any point in adding a test for one image?

Reply:
Normally I like tests for single and multiple values. So if the two images test fails, we can check if the single one passed if yes, we know it is problem affecting just the multiple case logic. If the single case also fails we know that it is a problem affecting the transform in general.